iphone/Maps/Classes/CustomViews/NavigationDashboard/Views/RoutePreview/MWMTaxiPreviewDataSource.mm

```diff
@@ -168,7 +176,18 @@ - (NSURL *)taxiURL;
 
   auto const index = [self.collectionView indexPathsForVisibleItems].firstObject.row;
   auto const productId = m_products[index].m_productId;
-  auto const links = Api::GetRideRequestLinks(productId, m_from, m_to);
+  RideRequestLinks links;
+  network_policy::CallPartnersApi(
+      [self, &productId, &links](platform::NetworkPolicy const &canUseNetwork) {
```

Comment:
Is it save to capture links by reference in callback? As far as I know, CallPartnersApi may put the callback on the end of the UI queue, to links won't be available at the call time.